.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Short description
+I have:
+- [ ] There is a corresponding issue for this pull request.
+- [ ] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
+- [ ] There is only strictly only one commit per issue.
+
+### For the reviewers
+I have:
+- [ ] Reviewed this pull request by an authorized contributor.
+- [ ] The reviewer is assigned to the pull request.
```

Comment:
It works if we all have a common practice and common vision. We need to align on that.